<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chart</class>
 <widget class="QWidget" name="Chart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="ButtonAxis">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>显示坐标轴</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonColumn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>三维柱状图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonDot">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>三维散点图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonPolyline">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>三维折线图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonPie">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>三维饼状图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonCurve">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>三维曲线图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonRadar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>三维雷达图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonParallel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>三维平行坐标图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonMark">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>240</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>三维空间标注</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonExit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
